src/features/movies/moviesSlice.js

```diff
@@ -26,20 +26,20 @@ const moviesSlice = createSlice({
     },
     fetchGenresSuccess: (state, { payload: genres }) => {
       state.genres = genres;
-      state.loading = false;
+      state.loading = true;
     },
     fetchGenresError: (state) => {
       state.loading = false;
     },
-    fetchMoviesByName: (state, { payload: query }) => {
+    fetchMoviesByQuery: (state, { payload: query }) => {
```

Comment:
to pole searchQuery w state.movies będzie nam potrzebne do aktualizowania nagłówków podczas wyszukiwania